server/analyze/analyze.py

```diff
@@ -64,10 +70,40 @@ def get_uptrend():
     return jsonify(res)
     # return json.dumps(res)
 
+@analyze.route("/getContentLength", methods=["GET"])
+def get_content_length():
+    current_app.logger.info(f"content length")
+    return "26000"
 
 @analyze.route("/options", methods=["GET"])
 def get_options():
     current_app.logger.info(f"analyzing data")
-    optionTrend = OptionTrend()
-    res = optionTrend.get_option_trend("equities")
-    return jsonify(list(res))
+    mode = "equities"
+    tickers = ['NIFTY','BANKNIFTY','FINNIFTY'] if mode == "indices" else get_nse_response(nse.equities_url)
```

Comment:
Always false

Reply:
yes as of now, changes will be made for indices